platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/attribute/StoredFileInput.java

```diff
@@ -54,6 +55,13 @@ protected void doSetDisabled(boolean disabled) {
 		uploadDiv.setDisabled(disabled);
 	}
 
+	@Override
+	public IEmfInput<AGStoredFile> appendLabel(IDisplayString label) {
+
+		//Not supported
```

Comment:
IMO, the comment looks very ugly ;)
```suggestion
		// no support for labels
```